geode-assembly/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/StopLocatorCommandDUnitTest.java

```diff
@@ -183,7 +182,7 @@ public void testWithMemberID() {
   }
 
   private void waitForCommandToSucceed(String command) {
-    await().atMost(5, TimeUnit.MINUTES).untilAsserted(() -> {
+    GeodeAwaitility.await().untilAsserted(() -> {
```

Comment:
I would prefer to get rid of the scoping here. (But I wouldn't hold up the merge for that.)